integration/Dockerfile

```diff
@@ -1,8 +1,11 @@
 FROM python:3.6
 
+WORKDIR /opt/cook/integration
+COPY requirements.txt /opt/cook/integration
```

Comment:
Why did this move? Can we remove the other WORKDIR line?

Reply:
This moved so that we have the latest copy of `requirements.txt` when running `pip` below.

The idea is that `requirements.txt` probably changes much less frequently than the rest of the test code in the directory, so moving this and the `pip` command up gives us more layers that can remain unchanged on a rebuild.

Reply:
Makes sense, but to be fair whenever you actually run the tests, they're mounted in a volume so there's no need to build the container unless the requirements.txt file changed.